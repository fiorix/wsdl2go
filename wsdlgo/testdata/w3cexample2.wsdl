<definitions>

<message name="newTermValues">
  <part name="term" type="xs:string"/>
  <part name="value" type="xs:string"/>
</message>

<portType name="glossaryTerms">
  <operation name="setTerm">
    <input name="newTerm" message="newTermValues"/>
  </operation>
</portType >

</definitions>
