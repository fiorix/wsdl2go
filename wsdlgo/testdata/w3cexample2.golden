package internal

import (
	"context"
	"errors"
)

// DoNothing was auto-generated from WSDL.
func DoNothing(ctx context.Context) error {
	return errors.New("not implemented")
}

// SetNothing was auto-generated from WSDL.
func SetNothing(ctx context.Context) error {
	return errors.New("not implemented")
}

// SetTerm was auto-generated from WSDL.
func SetTerm(ctx context.Context, term string, value string) error {
	return errors.New("not implemented")
}
